enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/discovery/Cluster.java

```diff
@@ -174,21 +174,37 @@ public ReadReplica addReadReplicaWithIdAndMonitors( @SuppressWarnings( "SamePara
         return member;
     }
 
-    public void shutdown() throws ExecutionException, InterruptedException
+    public void shutdown()
     {
-        shutdownReadReplicas();
-        shutdownCoreMembers();
+        try ( ErrorHandler errorHandler = new ErrorHandler( "Error when trying to shutdown cluster" ) )
+        {
+            shutdownCoreMembers( errorHandler );
```

Comment:
You switched the order of cores and read replicas here.

Reply:
Thanks. I'll update